discussion/app/controllers/CommentsController.scala

```diff
@@ -144,11 +144,10 @@ class CommentsController(csrfConfig: CSRFConfig, val discussionApi: DiscussionAp
           RevalidatableResult.Ok(views.html.discussionComments.discussionPage(page))
         }
       }
+    } recover {
+      case NonFatal(e) =>
+        InternalServerError(s"Discussion $key cannot be retrieved, has error: ${e.getMessage}")
```

Comment:
@johnduffell ah okay. I didn't realise logs were so easy to view in Frontend. Okay great - will update.